Gemfile

```diff
@@ -47,7 +47,3 @@ end
 group :deploy do
   gem "inquirer"
 end
-
-source "https://artifactory-internal.ps.chef.co/artifactory/api/gems/omnibus-gems-local/" do
```

Comment:
So, we want our builds to source from Artifactory. To continue to make this effect happen, we need to leave this stanza in place. However, to make it consumable by the public (ie not break CINC auditor), we need to conditionalize this under an environment variable. I'm going to propose CHEF_USE_INTERNAL_GEMSOURCES. 

I'll push a change to that effect.